cli/tests/unit/flash_test.ts

```diff
@@ -301,51 +301,6 @@ Deno.test(
   },
 );
 
-Deno.test(
-  { permissions: { net: true } },
-  async function httpReadHeadersAfterClose() {
```

Comment:
This is no longer necessary. `Deno.serveHttp()` returns properly formed response (ie. it doesn't do lazy loading of data), so headers are still available after the connection has been closed. This will allow us to remove some code in `ext/fetch/23_request.js`